gammapy/estimators/map/ts.py

```diff
@@ -166,7 +166,8 @@ class TSMapEstimator(Estimator, parallel.ParallelMixin):
 
     References
     ----------
-    [Stewart2009]_
+    .. [Stewart2009]_ `Stewart (2009), “Maximum-likelihood detection of sources among Poissonian noise”
```

Comment:
Why do you keep the "[Stewart2009]_" here, where in other python script you have removed the "[REF]" ?